src/Symfony/Bridge/Twig/Extension/HtmlSanitizerExtension.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Psr\Container\ContainerInterface;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * @author Titouan Galopin <tgalopin@gmail.com>
+ */
+final class HtmlSanitizerExtension extends AbstractExtension
+{
+    public function __construct(
+        private ContainerInterface $sanitizers,
+        string $defaultName = 'default',
```

Comment:
why making the default name configurable here if you remove it from the FrameworkBundle configuration ?

Reply:
for decoupling

Reply:
but then, why not keeping the default config configurable in FrameworkBundle ?

Reply:
What we need here is a default name that we can rely on. By making the default name configurable, the current way to configure html_sanitizer makes it hard for other bundles (twig-bundle here) to know what the default name is. Eg here I would have to make the bundle read the config settings of framework-bundle.

I solved this issue by making the default name non-configurable - aka I'm creating a convention: the default name is "default", and this "default" sanitizer is the one aliased to the `html_sanitizer` service, or to the non-named autowiring alias.

Does that make sense to you? Would you have a better idea?

Reply:
fine with me